ckanext/datagovuk/action/get.py

```diff
@@ -0,0 +1,65 @@
+import ckan.logic.schema
+from ckan.plugins.toolkit import (
+    abort,
+    check_access,
+    navl_validate,
+    ValidationError,
+    get_action,
+)
+import ckan.lib.dictization.model_save as model_save
+import ckan.lib.dictization.model_dictize as model_dictize
+
+from ckan.model import User
+
+
+log = __import__('logging').getLogger(__name__)
+
+
+# defined as they are in ckan/action/create.py to save further hacks to the
+# function copied from there
+_check_access = check_access
+_validate = navl_validate
+
+
+def user_auth(context, data_dict):
+    '''Authenticates a user
+
+    You must be a system administrator to authenticate users.
+
+    :param email: the email address of the user
```

Comment:
Yeah it'd just be to aid people transitioning from the old system, to continue logging in as they have done before. So I didn't see it as relevant to people registering afresh on the new system.